### ISSUE TITLE
Quenstion about sen_r in network,py

### ISSUE THREAD

**Issue by jaysoon** [thunlp/TensorFlow-NRE#19]
In Lin's paper, formula (7) computes attention of setence-level, in which e_i "scores how well the input sentence x_i and the predict relation r matches" and **r** of formula (8) "indicates the representation of relation r".
So,  I think r is the vector of target relation for a pair of entity。But in network.py, sen_r is initialized by random and shared by all train samples. I'm puzzled about it.
Did I misunderstand? How to understand r and sen_r？
Thanks.

**Comment by iamzww** [comment]
It indeed confused me as well. Have you figured it out, Jaysoon?

**Comment by iamzww** [comment]
The author have made a reply in #12  
"In the code, at present, I use the single query vector method for the sentence level attention. During the experiments, I also tried the multi-label query vectors method as described in the paper, but it turned out that it didn't achieve as good results as the single query vector method, which is also quite confusing for me."

**Comment by iamzww** [comment]
The author had made a reply in #12
"In the code, at present, I use the single query vector method for the sentence level attention. During the experiments, I also tried the multi-label query vectors method as described in the paper, but it turned out that it didn't achieve as good results as the single query vector method, which is also quite confusing for me."
